homeassistant/components/asuswrt/sensor.py

```diff
@@ -67,6 +72,21 @@
         SENSOR_FACTOR: 1000000000,
         SENSOR_ICON: "mdi:upload",
     },
+    SENSORS_LOAD_AVG[0]: {
```

Comment:
These could be considered less commonly used sensors. Should we disable these by default?

Reply:
All sensors with exception of the `device counter` are disabled by default:

```
    @property
    def entity_registry_enabled_default(self) -> bool:
        """Return if the entity should be enabled when first added to the entity registry."""
        return self._sensor_def.get(SENSOR_DEFAULT_ENABLED, False)
```